pkgs/applications/video/mpv/default.nix

```diff
@@ -151,18 +111,13 @@ stdenv'.mkDerivation (finalAttrs: {
     hash = "sha256-BOGh+QBTO7hrHohh+RqjSF8eHQH8jVBPjG/k4eyFaaM=";
   };
 
-  patches = [
-    # Fix build with Darwin SDK 11
-    ./0001-fix-darwin-build.patch
```

Comment:
Was this file removed from Nixpkgs repo?

Reply:
I intended to, but it’s possible it got lost in rebasing. Will open a PR soon if nobody beats me to it (currently racing against time for the 24.11 freeze).